src/Compilers/VisualBasic/Portable/Compilation/MethodCompiler.vb

```diff
@@ -847,9 +854,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                                          closureDebugInfo:=ImmutableArray(Of ClosureDebugInfo).Empty,
                                                          stateMachineTypeOpt:=Nothing,
                                                          variableSlotAllocatorOpt:=Nothing,
-                                                         debugDocumentProvider:=If(_moduleBeingBuiltOpt?.EmitOptions.EmitTestCoverageData, _debugDocumentProvider, Nothing),
+                                                         debugDocumentProvider:=If(_emitTestCoverageData, _debugDocumentProvider, Nothing),
                                                          diagnostics:=diagnosticsThisMethod,
                                                          emittingPdb:=False,
+                                                         emitTestCoverageData:=False,
```

Comment:
I was thinking we shouldn't need test coverage information on private implementation details. But I think you're correct: it is safer to keep whatever previous behavior was there.